pcdsdevices/pseudopos.py

```diff
@@ -221,6 +230,35 @@ def set_current_position(self, position):
         for motor, pos in zip(self._real, real_pos):
             motor.set_current_position(pos)
 
+    def _our_move_check(self, **kwargs):
+        """
+        If a different session moves this device, it's not our move!
+
+        This callback will set the self._my_move flag to False if the setpoint
+        updates far apart in time from our last move.
+
+        This will stop this instance from updating the notepad_readback
+        signals.
+        """
+        if time.monotonic() - self._move_time > self._my_move_timeout:
+            self._my_move = False
+
+    def _sub_our_move_checks(self):
+        """
+        Set up subscriptions for self._our_move_check
+        """
+        for positioner in self._pseudo:
+            try:
+                signal = getattr(positioner, 'notepad_setpoint', None)
+                if signal is None:
+                    continue
+                signal.subscribe(self._our_move_check)
+            except Exception as ex:
+                self.log.debug(
+                    'Failed to set up _our_move_check subscriptions.',
+                    exc_info=ex,
```

Comment:
Haha one last small thing on self-review
This is a bool kwarg and I'm passing an exception instance
This "works" but feels wrong

Reply:
TIL this actually also optionally takes an exception instance which in this case does the same thing but in other cases may not

Reply:
Right, for reference - https://docs.python.org/3/library/logging.html
```
Changed in version 3.5: The exc_info parameter can now accept exception instances.
```